src/mame/konami/hyperspt.cpp

```diff
@@ -848,6 +846,8 @@ ROM_START( hpolym84 )
 	ROM_LOAD( "c08",          0x0000, 0x2000, CRC(e8f8ea78) SHA1(8d37818e5a2740c96696f37996f2a3f870386690) )
 ROM_END
 
+/* Tecfri launched a Road Fighter PCB (probaby bootleg) with exactly the same software as the 'roadf'
+   set, but with ROMs 'J19_E14', 'G19_E18', 'A14_E26' and 'C14_E22' splitted on half capacity EPROMs. */
```

Comment:
What do you mean by “launched”?  They held a launch event with a picture of it?  Or are you saying they manufactured/sold it?  Also, the past tense of “split” is “split” – there is no “splitted”.

Reply:
Thanks. Fixed